### PR TITLE
VLCMediaPlayer: Update return value documentation

### PR DESCRIPTION
**Pull request by Mikanbu**
Updates the following methods documentation to mirror libvlc's:
- currentVideoTrackIndex
- currentVideoSubTitleIndex
- currentChapterIndex
- currentTitleIndex
- currentAudioTrackIndex
- updateViewpoint

### CODE REVIEW

File: Headers/Public/VLCMediaPlayer.h [hunk 1]
```diff
@@ -348,7 +348,7 @@ extern NSString * VLCMediaPlayerStateToString(VLCMediaPlayerState state);
 /**
  * Return the current video track index
  *
- * \return 0 if none is set.
+ * \return current video track index, -1 if none or no active input
```
Comment (fkuehne) [inline Headers/Public/VLCMediaPlayer.h]: replace "input" by "media track" to make this conceptually more understandable

File: Headers/Public/VLCMediaPlayer.h [hunk 2]
```diff
@@ -439,8 +439,8 @@ typedef NS_ENUM(unsigned, VLCMediaPlaybackSlaveType)
  */
 
 /**
- * Return the current chapter index, or
- * \return NSNotFound if none is set.
+ * Return the current chapter index
+ * \return current chapter index or -1 if there is no media
```
Comment (fkuehne) [inline Headers/Public/VLCMediaPlayer.h]: not "no media" but "no chapter"

File: Headers/Public/VLCMediaPlayer.h [hunk 3]
```diff
@@ -545,7 +545,7 @@ extern NSString *const VLCTitleDescriptionIsMenu;
 /**
  * Return the current audio track index
  *
- * \return 0 if none is set.
+ * \return current audio track index, -1 if none or no active input
```
Comment (fkuehne) [inline Headers/Public/VLCMediaPlayer.h]: "input" -> "media track"

**Review by fkuehne — COMMENTED** [review]
Nice work. Small nit-picks inside.

File: Headers/Public/VLCMediaPlayer.h (hunk 2 above)
Comment (Mikanbu) [reply Headers/Public/VLCMediaPlayer.h]: I wanted it to be in sync with [libvlc](https://gyazo.com/06ed9c2c7b8f0e02a554f9a5600bfb3f).
 I do not mind changing it but would that mean a modification on libvlc-side as well?

**Comment by Mikanbu** [comment]
This was merged, I will open a new PR if more modifications are needed!

**Pull request closed without merging**